apps/src/p5lab/utils.js

```diff
@@ -17,3 +19,34 @@ export function arrayEquals(a, b) {
 
   return true;
 }
+
+export function capitalizeFirstLetter(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+export function parsePathString(text) {
```

Comment:
Also - sorry for missing this earlier - could we name this something more specific to the sound field? Maybe `parseSoundPathString`?